pkg/http/server.go

```diff
@@ -185,6 +199,24 @@ func (s *Server) Serve() (err error) {
 	}
 
 	if s.hasScheme(schemeHTTPS) {
+		if s.LetsEncrypt {
+			if s.Domain == "" {
+				return errors.New("Need a domain name, use --domain <Domain Name>")
+			}
+			if !s.Production {
+				log.Warnf("Staging environment, use --production for production environment")
+			}
+			uDomain := strings.Replace(s.Domain, ".", "_", -1)
+			if s.existCertificate(uDomain) {
+				s.TLSCertificate = "./lets_encrypt/" + uDomain + ".crt"
```

Comment:
`./lets_encrypt/` should be a constant, you are using it in multiple places.

Reply:
I would go a step further and say this should be a configurable parameter. If we are going to be writing files we should give the end user a choice of where those files go.